lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/src/eos_helpers.cpp

```diff
@@ -243,28 +289,32 @@ namespace pew::eos
         return s_tempPathBuffer;
     }
 
-    void eos_create(config_legacy::EOSConfig eos_config)
+    void eos_create(const pew::eos::config::PlatformConfig& platform_config, const pew::eos::config::ProductConfig& product_config)
     {
         EOS_Platform_Options platform_options = { 0 };
         platform_options.ApiVersion = EOS_PLATFORM_OPTIONS_API_LATEST;
-        platform_options.bIsServer = eos_config.isServer;
-        platform_options.Flags = eos_config.flags;
+        platform_options.bIsServer = platform_config.is_server;
+        platform_options.Flags = platform_config.platform_options_flags;
         platform_options.CacheDirectory = GetCacheDirectory();
 
-        platform_options.EncryptionKey = eos_config.encryptionKey.length() > 0 ? eos_config.encryptionKey.c_str() : nullptr;
-        platform_options.OverrideCountryCode = eos_config.overrideCountryCode.length() > 0 ? eos_config.overrideCountryCode.c_str() : nullptr;
-        platform_options.OverrideLocaleCode = eos_config.overrideLocaleCode.length() > 0 ? eos_config.overrideLocaleCode.c_str() : nullptr;
-        platform_options.ProductId = eos_config.productID.c_str();
-        platform_options.SandboxId = eos_config.sandboxID.c_str();
-        platform_options.DeploymentId = eos_config.deploymentID.c_str();
-        platform_options.ClientCredentials.ClientId = eos_config.clientID.c_str();
-        platform_options.ClientCredentials.ClientSecret = eos_config.clientSecret.c_str();
+        platform_options.EncryptionKey = platform_config.client_credentials.encryption_key.c_str();
+
+        // TODO: Determine why now this cannot be empty.
+        platform_options.OverrideCountryCode = "US";// platform_config.overrideCountryCode.c_str();
+        platform_options.OverrideLocaleCode = "en-US"; // platform_config.overrideLocaleCode.c_str();
```

Comment:
Again - do the thing. Maybe they can't be empty strings, but they can be set to `nullptr`? This would be consistent with the prior implementation.

Reply:
Solution was implemented so that if either of the strings for `overrideLocaleCode` or `overrideCountryCode` is empty, the respective parameters given to create the platform are `nullptr`. This restores original implementation prior to this PR - so this conversation is being marked as resolved.